docs/conf.py

```diff
@@ -1,7 +1,8 @@
-import sphinx_rtd_theme
 from recommonmark.parser import CommonMarkParser
 from recommonmark.transform import AutoStructify
 
+import sphinx_rtd_theme
```

Comment:
The existing order was correct here. The reason isort in Vagrant will be telling you otherwise, is that by default the docs requirements are not installed in Vagrant, so isort can't correctly categorise them as first/third party.

Our options are to:
1. ignore for now and wait to see if this file ends up being removed as part of the switch to mkdocs
2. exclude this file from isort
3. install the docs dependencies by default in Vagrant (slightly slower initial provision time, but probably still a drop in the ocean)

I'm open to any of the above :-) (Just in a separate PR)